src/main/java/week2/LeftRotation.java

```diff
@@ -0,0 +1,44 @@
+package week2;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class LeftRotation {
+
+    private static void rotate(int[] a, int d) {
+        while(d>0) {
+            int start = a[0];
+            for(int i=0; i<a.length-1; i++) {
```

Comment:
Try to improve it to O(n).
explore discussion section or editorial for some exiting  and better solution

Reply:
Ok Will try to see some better solution from discussions